batdata/data.py

```diff
@@ -213,12 +212,19 @@ def from_batdata_hdf(cls, path_or_buf: Union[str, Path, HDFStore], subsets: Opti
             read_all = True
 
         data = {}
-        for key in subsets:
-            if key not in _subsets:
-                raise ValueError(f'Unknown subset: {key}')
+        for subset in subsets:
+            # Throw error if user provides an unknown subset name
+            if subset not in _subsets:
+                raise ValueError(f'Unknown subset: {subset}')
+
+            # Prepend the prefix
+            if prefix is not None:
+                key = f'{prefix}-{subset}'
+            else:
```

Comment:
If we want to get all the cells stored in the hdf5 file, regardless of 'prefix', does this work? Or would it raise an error? 
Also, doesn't this require one to know all the prefixes that have been stored?

Reply:
Yea, it does require knowing the prefixes if they are used when saving the file. 

Maybe we should return an error if the prefix is not included in those found in the file?

Reply:
I'm going to add that warning message

Reply:
Done